pkg/config/pulse_server.go

```diff
@@ -23,9 +25,11 @@ func MewPulseServerConfig() *PulseServerConfig {
 // - "pulse-interval" sets the time between maestro instance pulses (in seconds) to indicate its liveness (default: 10 seconds).
 // - "check-interval" determines how often health checks are performed on maestro instances (in seconds) to maintain the active instance list (default: 10 seconds).
 // Instances not pulsing within the last three check intervals are considered as dead.
+// - "enable-consistent-hashing" enables consistent hashing for resource status update distribution among running maestro instances (default: false).
 func (c *PulseServerConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.Int64Var(&c.PulseInterval, "pulse-interval", c.PulseInterval, "Set the pulse interval for maestro instances (seconds) to indicate liveness (default: 10 seconds)")
 	fs.Int64Var(&c.CheckInterval, "check-interval", c.CheckInterval, "Set the interval for health checks on maestro instances (seconds) to maintain the active instance list (default: 10 seconds)")
+	fs.BoolVar(&c.EnableConsistentHashing, "enable-consistent-hashing", c.EnableConsistentHashing, "Enable consistent hashing for resource status update distribution among running maestro instances (default: false)")
```

Comment:
I think we should not expose detailed implementation as flag. Maybe just call it subscription-type, and by default it is shared. and it can also be something like broadcast

Reply:
changed to `subscription-type`, support `shared` (default value) and `broadcast`.